cloudferrylib/utils/file_proxy.py

```diff
@@ -38,18 +37,8 @@ def get_file_size(file_obj):
             size = file_obj.tell()
             file_obj.seek(curr)
             return size
-        except IOError as e:
-            if e.errno == errno.ESPIPE:
-                # Illegal seek. This means the file object
-                # is a pipe (e.g the user is trying
-                # to pipe image data to the client,
-                # echo testdata | bin/glance add blah...), or
-                # that file object is empty, or that a file-like
-                # object which doesn't support 'seek/tell' has
-                # been supplied.
-                return
-            else:
-                raise
+        except IOError:
+            return
 
 
 class ProgressView(object):
```

Comment:
Please fix progress view for ReCreateBootImage file_proxy.py:69 Copying image shows incorrect values:
12.1MB of 1.8MB 665%